content/updates/2018-02.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 2018.2.0
+date: 2018-03-07
+hidden: true
+image: TODO
+---
+
+# 2018.02 Changes
```

Comment:
Is "Changelog" better here?

Reply:
Fixed in 8a23f2662eead82ac09e7c23a7d1148a3c038387